backend/src/db/schemas/models.ts

```diff
@@ -205,5 +210,6 @@ export enum IdentityAuthMethod {
 export enum ProjectType {
   SecretManager = "secret-manager",
   CertificateManager = "cert-manager",
-  KMS = "kms"
+  KMS = "kms",
+  SSH = "ssh"
```

Comment:
I think naming the product as `ssh` could be troublesome. It feels too generic.

Reply:
I think this should be fine; this is the general category of product I'm seeing.

Unless you have suggestions?